### DISCUSSION TITLE
ST receipe in espnet2 missing MT init

### DISCUSSION THREAD

**Discussion by pratikkumar018** [espnet/espnet#4035] [Q&A]
I was previously using ST in espnet 1. There was enc-init & dec-init where we can initialize pretrained asr & MT models. But now in espnet I am only seeing one parameter as pretrained_asr . So now we cant directly initialize MT models or am I missing anything?

**Comment by sw005320** [comment]
Thanks for the discussion.
We're actually working on it.

**Reply by pratikkumar018** [reply]
Is the current recipe usable without this feature in espnet2?

**Reply by ftshijt** [reply]
Yeah, it is. As noted in fisher_callhome_spanish recipe, we could reach comparable performances to espnet1. Since we do not have support for MT task yet, we did not publish mt_init method for the current template version. But we are working on it, stay tuned!

**Reply by pratikkumar018** [reply]
Thanks for the info.

**Comment by sw005320** [comment]
@ftshijt
